sensors/test_deps.go

```diff
@@ -75,7 +75,7 @@ func getWarmingUpLidar() *inject.Camera {
 		return nil, transform.NewNoIntrinsicsError("")
 	}
 	cam.PropertiesFunc = func(ctx context.Context) (camera.Properties, error) {
-		return camera.Properties{}, nil
+		return camera.Properties{SupportsPCD: true}, nil
```

Comment:
Here and below, we could leave the PropertiesFunc returning an empty Properties struct unless a test actually requires it to return `SupportsPCD: true`

Reply:
Every time we call NewLidar on a lidar we are checking that it supports the properties, so I think we wouldn't be able to do any testing with these injected lidars if they do not fulfill the properties requirement